setup.py

```diff
@@ -69,6 +69,32 @@ def remove_specific_requirements(reqs):
     return updated_reqs
 
 
+def fetch_files_from_folder(folder):
+    options = glob(f"{folder}/**", recursive=True)
+    data_files = []
+    for option in options:
+        if os.path.isdir(option):
+            files = []
+            for f in glob(os.path.join(option, "*")):
+                if os.path.isfile(f):
+                    files.append(f)
+                data_files += files
+    return data_files
+
+
+def fetch_package_data():
```

Comment:
Just curious .. Why each files from folders need to be added?  Maybe some comments will be good for future maintainers.